test/xpu_api/random/statistics_tests/extreme_value_distribution_dp_portion_test.pass.cpp

```diff
@@ -0,0 +1,183 @@
+// -*- C++ -*-
+//===-- extreme_value_distribution_dp_tests_set_portion_test.cpp ---------------------------------===//
```

Comment:
Please, check that new filenames are matched with the comments.
Also, a suggestion: maybe we can shorted `distribution` to `distr` as well, as it still seems very long